ts/components/screens/OperationResultScreenContent.tsx

```diff
@@ -15,7 +15,7 @@ import { Platform, SafeAreaView, StyleSheet, View } from "react-native";
 import { ScrollView } from "react-native-gesture-handler";
 import { LabelSmall } from "../core/typography/LabelSmall";
 
-type OperationResultScreenContent = {
+type OperationResultScreenContentProps = WithTestID<{
```

Comment:
the type needs to be updated in the tests aswell

Reply:
Fixed https://github.com/pagopa/io-app/pull/5224/commits/133129ab11764412c3c7bdd16215acb6b15a8648